### ISSUE TITLE
Link colour on white has insufficient contrast

### ISSUE THREAD

**Issue by anthonydillon** [ubuntudesign/vanilla-framework#374]
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.65:1. Recommendation: change text colour to #d6410d.

**Comment by anthonydillon** [comment]
What information do you need?

**Comment by barrymcgee** [comment]
@anthonydillon - same as @yaili asked in #373, you mention a conformance level but don't specify which conformance level? 

Is that an output message from an auditing tool? If so, which one?

**Comment by anthonydillon** [comment]
It was reported by a tool I use [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/). 

This failed WCAG2AA conformance. 

I am concerned that we might not be willing to fix issues relating to AAA standard. Even if the fix is seemly straight forward. I think we should aim to fix as many as possible.

**Comment by barrymcgee** [comment]
We should be absolutely willing to resolve issues relating to accessibility conformance but where we are at the moment with the project as a whole I think we need to be careful of bikeshedding. 

The stable branch of Vanilla is probably very far off AA conformance, never mind AAA, and given the limited time and man power we have to overhaul the project for v1, I feel the focus should remain on v1.

In the meantime it's important to follow the issue template providing the information needed to replicate the bug so that anyone with a spare hour can nick off the low hanging fruit like this issue... 👍

**Comment by barrymcgee** [comment]
Resolved by #449
